src/atoms/icon1/Icon.html

```diff
@@ -0,0 +1,12 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24px"
+  height="24px"
+  class="a-icon"
+  viewBox="0 0 24 24"
+  :aria-labelledby="iconTitle"
+  role="presentation"
```

Comment:
Afaik we should use `role="img"` + `<title>` when the icon 'matters'.
Otherwise there should be `role="presentation"` without `<title>`.
So I suggest to add a prop with default string 'img' and use for that.
You can ask Ania to make sure

Reply:
done